lib/health.js

```diff
@@ -1,82 +1,91 @@
 const log = require('metalogger')();
-const checks = {};
 
 const STATUS_ERR = 'fail';
 const STATUS_OK   = 'pass';
 const STATUS_WARN = 'warn';
 
-module.exports = function(opts) {
+class HealthCheck {
 
-  const healthURI = process.env.NODE_HEALTH_ENDPOINT_PATH || "/health";
-  return async (req, res, next) => {
+  constructor(opts) {
+    this.checks = [];
+    this.opts = opts || {};
+  }
 
-    const requestPath = req.originalUrl;
+  handler() {
```

Comment:
Since you anticipate supporting more than express, why not call this `express`, or add an argument for what type of handler to return?

Reply:
That's a great point, and also express should be a dev dependency, not a direct dependency...

Fixing...